prov/usnic/src/usdf_av.c

```diff
@@ -646,6 +646,9 @@ usdf_av_open(struct fid_domain *domain, struct fi_av_attr *attr,
 		return -FI_ENOSYS;
 	}
 
+	if (attr->name)
+		return -FI_ENOSYS;
```

Comment:
@jsquyres Should we be returning `-FI_ENOSYS` or `-FI_EINVAL`?...I'm thinking `ENOSYS`.